### PR TITLE
Workflow Submission:  Change row components to column components for faster turbo updates

### PR DESCRIPTION
**Pull request by joshsadam**
## What does this PR do and why?
_Describe in detail what your merge request does and why._

Refactor of the samplesheet component to allow turbo updates of only a single column of the table instead of updating n number of cells separately.

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

No real changes, now looks like:

**Light**
image

**Loading State**
loading

**Dark**
image

## How to set up and validate locally
_Numbered steps to set up and validate the change are strongly suggested._

Try running a pipeline, there should be no changes.

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.

- [X] I have evaluated the [PR acceptance checklist](https://phac-nml.github.io/irida-next/docs/development/development_processes/code_review#acceptance-checklist) for this PR.

### CODE REVIEW

File: app/components/nextflow/samplesheet_component.html.erb [hunk 1]
```diff
@@ -1,70 +1,52 @@
...
-          <%= render Nextflow::Samplesheet::RowComponent.new(
-            sample,
-            index,
-            properties,
-            required_properties
-          ) %>
-        <% end %>
-      </tbody>
-    </table>
+        <span class="flex items-center p-4 space-x-4">
```
Comment (ericenns) [inline app/components/nextflow/samplesheet_component.html.erb:43]: **nitpick (blocking):** This span is missing a closing span tag.

Comment (joshsadam) [reply app/components/nextflow/samplesheet_component.html.erb:43]: Fixed in e9a2872, thanks

**Review by ericenns — CHANGES_REQUESTED** [review]
This is looking good, minus a simple css background bug
 
image

**Review by ericenns — APPROVED** [review]
Looks good to me!

**Review by deepsidhu85 — APPROVED** [review]
Looks good to me!

**Pull request merged**